FoodTracker/FoodTracker/Coordinators/ApplicationCoordinator.h

```diff
@@ -0,0 +1,14 @@
+//
+//  ApplicationCoordinator.h
+//  FoodTracker
+//
+//  Created by hello on 25/07/2018.
+//  Copyright © 2018 digits. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "Coordinator.h"
+#import "MealTableViewCoordinator.h"
+@interface ApplicationCoordinator: NSObject <Coordinator>
+- (instancetype) init:(UIWindow *)window;
```

Comment:
use [apple convention](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/Conventions/Conventions.html), in that case `initWithWindow:(UIWindow *)window`